### DISCUSSION TITLE
Should we add conduit as its own class?

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [PMEAL/OpenPNM#2191] [Ideas]
Just throwing out ideas, and obviously not for serious consideration for v3, but just saying: I just realized that we have "circles_and_cylinders" method in 4 different places: 

- diffusive_size_factors.py
- hydraulic_size_factors.py
- throat_length.py
- conduit_lenghts.py

It made me wonder maybe "conduits" are suitable to be a separate class...

**Comment by jgostick** [comment]
you'll need to write a small mock-up...I can't quite see the solution you have in mind.
